### PR TITLE
Add stylint to project

### PR DESCRIPTION
**Pull request by 3BenLee**
Changes:
1) added stylelint package
2) connected stylelint with the travis and circle ci pipeline
3) set basic initial style sheet lint rules (the rules can be customized in the future)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "lint": "eslint 'src/**/*.ts?(x)'",
     "lint:fix": "eslint --fix 'src/**/*.ts?(x)'",
+    "stylelint": "stylelint 'src/**/style.less'",
```
Comment (davidnguyen179) [inline package.json]: could it be "src/**/*.less"?

File: .stylelintrc.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "stylelint-config-recommended",
```
Comment (davidnguyen179) [inline .stylelintrc.json]: Does it one support the "kebab-case"?

File: package.json (hunk 1 above)
Comment (3BenLee) [reply package.json]: Done.

File: .stylelintrc.json (hunk 2 above)
Comment (3BenLee) [reply .stylelintrc.json]: Done. You have to do it using regex.  

https://stackoverflow.com/questions/58499638/integrate-stylelint-with-angular-build-and-make-it-failed-on-error

Comment (davidnguyen179) [reply .stylelintrc.json]: you would take a look at this one

https://github.com/stylelint/stylelint/blob/master/docs/user-guide/faq.md#how-do-i-configure-the--pattern-rules-for-common-css-naming-conventions-like-kebab-case

Comment (3BenLee) [reply .stylelintrc.json]: Yeah, I used the same regex in the link you shared:
```"selector-class-pattern": ["^([a-z][a-z0-9]*)(-[a-z0-9]+)*$", {
      "resolveNestedSelectors": true
    }]```

Comment (davidnguyen179) [reply .stylelintrc.json]: Also help me to add to this file. It's the "github workflow"

https://github.com/davidnguyen179/storybook-wdio/blob/master/.github/workflows/ci.yml

Comment (3BenLee) [reply .stylelintrc.json]: Done.

**Pull request merged**